src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs

```diff
@@ -134,6 +134,16 @@ public override void ExecuteCmdlet()
                 databaseAccountUpdateParameters.IpRules = base.PopulateIpRules(IpRule);
             }
 
+            if (ServerVersion != null)
```

Comment:
this can be a shared code in NewOrUpdateAzCosmosDBAccount class

Reply:
It's hard to share much more than the null guard here since the parameters objects are different.